File: INI4/ini4.go
package main

import (
	"fmt"
	"os"
	"strconv"
)

func main() {
	if len(os.Args) < 3 {
		fmt.Printf("Usage: %v <num1> <num2>\n", os.Args[0])
		os.Exit(1)
	}

	num1, err := strconv.Atoi(os.Args[1])
	if err != nil {
		fmt.Println("error:", err)
	}
	num2, err := strconv.Atoi(os.Args[2])
	if err != nil {
		fmt.Println("error:", err)
	}

	var sum int = 0
	for i:=num1; i<=num2; i++ {
		if i % 2 == 1 {
			sum += i
		}
	}

	fmt.Println(sum)
}

File: INI4/ini4.c
#include <stdio.h>
#include <stdlib.h>

// print sum of odd numbers from given range

int main(int argc, char **argv) {
    if (argc < 3) {
        printf("Usage: %s <num1> <num2>\n", argv[0]);
        return 1;
    }

    int num1 = atoi(argv[1]);
    int num2 = atoi(argv[2]);

    unsigned int sum=0;
    for (int i=num1; i<=num2; i++)
        if (i & 1) sum += i;

    printf("%u\n", sum);
    return 0;
}
